tools/encodings/GF Glyph Sets/Cyrillic/filter lists/uni names/plus_unique-glyphs.txt

```diff
@@ -189,4 +191,4 @@ tugrik
 descender-cy
```

Comment:
Putting these into RECOMMENDED.md with the text:

### Cyrillic component building

These glyphs are used for component building and are non-exportable. Don't forget to set the non-export tag.

```
descender-cy    
descender-cy.case
verticalbar-cy
verticalbar-cy.case